jobs/kola-gcp.Jenkinsfile

```diff
@@ -0,0 +1,66 @@
+@Library('github.com/coreos/coreos-ci-lib') _
+
+def streams
+node {
+    checkout scm
+    streams = load("streams.groovy")
+    pod = readFile(file: "manifests/pod.yaml")
+}
+
+properties([
+    pipelineTriggers([]),
+    parameters([
+      choice(name: 'STREAM',
+             // list devel first so that it's the default choice
+             choices: (streams.development + streams.production + streams.mechanical),
+             description: 'Fedora CoreOS stream to test'),
+      string(name: 'VERSION',
+             description: 'Fedora CoreOS Build ID to test',
+             defaultValue: '',
+             trim: true),
+      string(name: 'S3_STREAM_DIR',
+             description: 'Fedora CoreOS S3 Stream Directory',
+             defaultValue: '',
+             trim: true),
+      string(name: 'COREOS_ASSEMBLER_IMAGE',
+             description: 'Override coreos-assembler image to use',
+             defaultValue: "coreos-assembler:master",
+             trim: true)
+    ])
+])
+
+currentBuild.description = "[${params.STREAM}] - ${params.VERSION}"
+
+def s3_stream_dir = params.S3_STREAM_DIR
+if (s3_stream_dir == "") {
+    s3_stream_dir = "fcos-builds/prod/streams/${params.STREAM}"
+}
+
+cosaPod(image: params.COREOS_ASSEMBLER_IMAGE, memory: "256Mi",
+        secrets: ["aws-fcos-builds-bot-config", "gcp-kola-tests-config"]) {
+
+    def gcp_image, gcp_image_project, gcp_project
+    stage('Fetch Metadata') {
+        shwrap("""
+        export AWS_CONFIG_FILE=/run/kubernetes/secrets/aws-fcos-builds-bot-config/config
+        cosa init --branch ${params.STREAM} https://github.com/coreos/fedora-coreos-config
+        cosa buildprep --ostree --build=${params.VERSION} s3://${s3_stream_dir}/builds
+        """)
+
+        def basearch = shwrapCapture("cosa basearch")
+        def meta = readJSON file: "builds/${params.VERSION}/${basearch}/meta.json"
+        if (meta.gcp.image) {
+            gcp_image = meta.gcp.image
+            gcp_image_project = meta.gcp.project
+        } else {
+            throw new Exception("No GCP image found in metadata for ${params.VERSION}")
+        }
+
+        // pick up the project to use from the config
+        def config = readJSON file: "/run/kubernetes/secrets/gcp-kola-tests-config/config"
+        gcp_project = config.project_id
+    }
+
+    fcosKola(cosaDir: env.WORKSPACE, parallel: 5,
+             extraArgs: "-p gce --build ${params.VERSION} --gce-project ${gcp_project} --gce-image project/${gcp_image_project}/global/images/${gcp_image} --gce-json-key=/run/kubernetes/secrets/gcp-kola-tests-config")
```

Comment:
any way to break this up into multiple lines? tends to make it easier to read when we have the `--` options on multiple lines like they were before.